shell/platform/android/io/flutter/app/FlutterFragmentActivity.java

```diff
@@ -128,6 +128,7 @@ protected void onPostResume() {
   // @Override - added in API level 23
```

Comment:
I believe we can also remove this comment because this is now using androidx support library so it should not matter when the API was added.

Reply:
Done.